frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/tests/setup.tsx

```diff
@@ -49,27 +49,23 @@ export interface SetupOpts {
   hasEnterprisePlugins?: boolean;
 }
 
-export async function setup(
-  {
-    props: {
-      resourceType = "dashboard",
-      resource = getMockResource(resourceType),
-      resourceParameters = [],
-      onUpdateEmbeddingParams = jest.fn(),
-      onUpdateEnableEmbedding = jest.fn(),
-    },
-    activeTab = "Overview",
-    hasEnterprisePlugins = false,
-    tokenFeatures = createMockTokenFeatures(),
-  }: {
-    props: Partial<StaticEmbedSetupPaneProps>;
-    activeTab?: "Overview" | "Parameters" | "Look and Feel";
-    hasEnterprisePlugins?: boolean;
-    tokenFeatures?: TokenFeatures;
-  } = {
-    props: {},
```

Comment:
remove this and add it to line 59 on the right instead since this doesn't seem to work. When I don't pass `props` the test fails.